wordpress-mobile/WordPress-Android
# Master edit post activity refactor

This is a master branch. All the code has already been reviewed and tested.

The most important PRs which were merged into this branch are
- https://github.com/wordpress-mobile/WordPress-Android/pull/10763
- https://github.com/wordpress-mobile/WordPress-Android/pull/10723

**Merge instructions:**
1. Merge FluxC's branch - https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1417
2. Create a new FluxC version/tag
3. Update the FluxC version in WPAndroid
4. Push the changes
5. Wait for Circle CI
6. Merge the PR

To test:
- See the most important PRs for test instructions - Vojta and I have tried to test the whole editor.

PR submission checklist:



cc @hypest @maxme @daniloercoli You mentioned you want to check and perhaps test the PR when it's ready.
